pkgs/applications/version-management/git-and-tools/tig/default.nix

```diff
@@ -3,11 +3,11 @@
 }:
 
 stdenv.mkDerivation rec {
-  name = "tig-2.2";
+  name = "tig-2.2.1";
 
   src = fetchurl {
-    url = "http://jonas.nitro.dk/tig/releases/${name}.tar.gz";
```

Comment:
Have you tried fetchFromGitHub, as it is now the preferred method when possible. When you do it, rev is most likely gonna be:
rev = "v${version}"; sometimes it is rev = version; according how the upstream lists their version